index.html

```diff
@@ -1,579 +1,998 @@
+      </section>
+    </section>
 
-        <p>
-Implementations MUST produce an error when an extension JSON-LD Context
-overrides the expanded URL for a term specified in this specification. To avoid
-the possibility of accidentally overriding terms, developers SHOULD scope their
-extensions. For example, the following extension scopes the new
-<code>PhotoStreamService</code> term so that it can only be used within the
-<code>service</code> property:
-        </p>
+    <section class="informative">
+      <h3>Representations</h3>
 
-        <pre class="example nohighlight" title="Scoping terms in a JSON-LD Context">
-{
-    "@context": {
-        <span class="highlight">"service": {
-            "@id": "https://w3id.org/did#service",
-            "@context": {
-                "PhotoStreamService": "https://example.com/vocab#PhotoStreamService"
-            }
-        }</span>
-    }
-}
-        </pre>
+      <p class="note">
+        See [[DID-CORE]] for normative requirements associated with developing
+        <a href="https://www.w3.org/TR/did-core/#representations">Representations</a>.
+      </p>
+
+      <p>
+        The [[DID-CORE]] specification describes representations and the
+        [[DID-SPEC-REGISTRIES]] supports the registration of an unbounded
+        number of alternative representations.
+      </p>
+
+      <p>Each representation has unique pros and cons.</p>
+
+      <p>
+        DID Methods are responsible for producing a DID Document in the
+        requested representation.
+      </p>
+
+      <p>
+        DID Method implementers may choose to return verification methods in
+        alternate formats for representations, for example, an implementer
+        might prefer to return verification material encoded with
+        <code>publicKeyJwk</code> for <code>application/did+json</code> and
+        <code>publicKeyBase58</code> for <code>application/did+ld+json</code>.
+      </p>
+
+      <p>
+        If a DID Method supports both <code>application/did+json</code> and
+        <code>application/did+ld+json</code> if it recommended that they both
+        include an <code>@context</code> and that both be capable of
+        supporting linked data proofs.
+      </p>
+
+      <p>
+        Avoid allowing arbitrary unknown properties into
+        <code>application/did+json</code> such as <code>__proto__</code> or
+        other characters that might be used to attack JSON parsing.
+      </p>
+
+      <p>
+        While <code>application/did+json</code> is very flexible, and allows
+        for arbitrary JSON, implementers are cautioned to implement security
+        in depth.
+      </p>
+
+      <p>
+        Implementers are cautioned to review
+        <a href="https://owasp.org/www-project-top-ten/">OWASP Top 10</a>
+      </p>
+
+      <p>
+        Implementers should confirm that each representation they decide to
+        support is interoperable with the same representation in other did
+        methods.
```

Comment:
```suggestion
        support is interoperable with the same representation in other DID
        Methods.
```
I no longer remember which "DID *" terms have an initial capital and which do not. I am at least confident that "DID" should always be capitalized in prose.